src/types/events-api.ts

```
@@ -0,0 +1,415 @@
+/**
+ * Type definitions for GitHub Events API
+ * Extends Octokit's official types with discriminated payload unions
+ * Includes Zod schemas for runtime validation
+ * https://docs.github.com/en/rest/activity/events
+ */
+
+import type { Endpoints } from "@octokit/types";
+import { z } from "zod";
```

ESLint

[error] Unable to resolve path to module 'zod'.

(import-x/no-unresolved)